test/elixir_sense/core/source_test.exs

```diff
@@ -561,4 +561,98 @@ defmodule ElixirSense.Core.SourceTest do
       assert which_struct(text_before(code, 5, 7)) == {Mod, [:field1]}
     end
   end
+
+  describe "alias syntax v1.2" do
+    test "single level" do
+      code = """
+      defmodule MyMod do
+        def my_func(par1) do
+          alias Mod.{
+      """
+
+      assert get_v12_module_prefix(code) == Mod
+    end
+
+    test "single level with submodule" do
+      code = """
+      defmodule MyMod do
+        def my_func(par1) do
+          require Mod.{Su
+      """
+
+      assert get_v12_module_prefix(code) == Mod
+    end
+
+    test "single level with submodules" do
+      code = """
+      defmodule MyMod do
+        def my_func(par1) do
+          use Mod.{Su.Bmod, Other
+      """
+
+      assert get_v12_module_prefix(code) == Mod
+    end
+
+    test "multi level with submodules" do
+      code = """
+      defmodule MyMod do
+        def my_func(par1) do
+          import Mod.Sub.{
+      """
+
+      assert get_v12_module_prefix(code) == Mod.Sub
+    end
+
+    test "__MODULE__ special form level with submodules" do
+      code = """
+      defmodule MyMod do
+        def my_func(par1) do
+          alias __MODULE__.{
+      """
+
+      assert get_v12_module_prefix(code) == :__MODULE__
```

Comment:
Will the code complete still work in this case? Or should it be replaced by `MyMod` somehow? In case it doesn't work, no worries for now, we can handle it later.

Reply:
Good point, it won't. I'll fix that case.

Reply:
It turned out to be more difficult than I expected. I'll do it in separate PR.